build.gradle

```diff
@@ -55,7 +55,7 @@ ext {
     zstdVersion = '1.5.5-2'
     awsSdkV2Version = '2.18.9'
     awsSdkV1Version = '1.12.337'
-    mongoCryptVersion = '1.8.0-alpha1'
+    mongoCryptVersion = '1.8.0'
```

Comment:
This hasn't been released yet (https://repo1.maven.org/maven2/org/mongodb/mongodb-crypt/).

Reply:
I assume that's why all the checks are failing.

Let's revert this change and do it separately, after we release 1.8.0 Java bindings.

Reply:
Reverted back to version 1.9.0-alpha1